packages/editor-ui/src/stores/users.store.test.ts

```diff
@@ -58,4 +63,29 @@ describe('users.store', () => {
 			expect(identify).toHaveBeenCalledWith('test-instance-id', mockUser.id);
 		});
 	});
+
+	describe('inviteUsers', () => {
+		it('should add pending user to the store', async () => {
+			const usersStore = useUsersStore();
+
+			inviteUsers.mockResolvedValueOnce([
+				{
+					user: { role: 'global:member', email: 'test@n8n.io', emailSent: true },
+				},
+			]);
+
+			await usersStore.inviteUsers([{ email: 'test@n8n.io', role: 'global:admin' }]);
+
+			expect(usersStore.allUsers[0]).toMatchObject(
+				expect.objectContaining({
+					role: 'global:member',
```

Comment:
```suggestion
					role: 'global:admin',
```

Reply:
Why do we need this?/

Reply:
On line 77 it says the role is `global:admin`. We can also change line 77 to `global:member`.